interfaces/PA-App/src/environments/environment.prod.ts.template.js

```diff
@@ -12,7 +12,7 @@ export const environment = {
   localStorage: ${process.env.NG_LOCAL_STORAGE || 'false'},
 
   // Extra:
-  subDirPath: '${process.env.NG_SUB_DIR_PATH}',
+  subDirPath: ${process.env.NG_SUB_DIR_PATH_PA || '\'/PA-app\''},
```

Comment:
This default value is not safe to use for the native-builds of the app(s).
+ The syntax shouldn't be what you think it should be. ;)